docs/about/features_index/taskrunner.rst

```diff
@@ -382,7 +382,7 @@ Importing the Workspace
 
     .. code-block:: console
 
-       $ fx collaborator create -n {COL_LABEL} -d {DATA_PATH:optional}
+       $ fx collaborator create -n {COL_LABEL} -d {DATA_PATH}

```

Comment:
In general, this should an optional parameter, to cover cases where the data path is already set in `data.yaml` as part of the FL plan creation

Reply:
In some examples (the ones modified by this PR), data path represents the shard number to be used for that collaborators. When we use the default value [default_data_path = f"data/{collaborator_name}"](https://github.com/securefederatedai/openfl/blob/develop/openfl/interface/collaborator.py#L140), an error is encountered when starting the collaborator is started. 
```
Data = {'c1': 'data/c1', 'c2': 'data/c2'}
           INFO     🧿 Starting a Collaborator Service.                      collaborator.py:99
           INFO     Building `src.dataloader.PyTorchMNISTInMemory` Module.   plan.py:228
EXCEPTION : invalid literal for int() with base 10: 'data/c1'
Traceback (most recent call last):
  File "/root/openfl-maestro/bin/fx", line 8, in <module>
    sys.exit(entry())
  File "/root/openfl-maestro/lib/python3.8/site-packages/openfl/interface/cli
.py", line 328, in entry
    error_handler(e)
  File "/root/openfl-maestro/lib/python3.8/site-packages/openfl/interface/cli
.py", line 246, in error_handler
    raise error
  File "/root/openfl-maestro/lib/python3.8/site-packages/openfl/interface/cli
.py", line 326, in entry
    cli(max_content_width=120)
  File "/root/openfl-maestro/lib/python3.8/site-packages/click/core.py", line
1157, in __call__
    return self.main(*args, **kwargs)
  File "/root/openfl-maestro/lib/python3.8/site-packages/click/core.py", line
1078, in main
    rv = self.invoke(ctx)
  File "/root/openfl-maestro/lib/python3.8/site-packages/click/core.py", line
1688, in invoke
    return _process_result(sub_ctx.command.invoke(sub_ctx))
  File "/root/openfl-maestro/lib/python3.8/site-packages/click/core.py", line
1688, in invoke
    return _process_result(sub_ctx.command.invoke(sub_ctx))
  File "/root/openfl-maestro/lib/python3.8/site-packages/click/core.py", line
1434, in invoke
    return ctx.invoke(self.callback, **ctx.params)
  File "/root/openfl-maestro/lib/python3.8/site-packages/click/core.py", line
783, in invoke
    return __callback(*args, **kwargs)
  File "/root/openfl-maestro/lib/python3.8/site-packages/openfl/interface/col
laborator.py", line 101, in start_
    plan.get_collaborator(collaborator_name).run()
  File "/root/openfl-maestro/lib/python3.8/site-packages/openfl/federated/pla
n/plan.py", line 601, in get_collaborator
    data_loader = self.get_data_loader(collaborator_name)
  File "/root/openfl-maestro/lib/python3.8/site-packages/openfl/federated/pla
n/plan.py", line 453, in get_data_loader
    self.loader_ = Plan.build(**defaults)
  File "/root/openfl-maestro/lib/python3.8/site-packages/openfl/federated/pla
n/plan.py", line 235, in build
    instance = getattr(module, class_name)(**settings)
  File "/home/akshay/akshay/src/dataloader.py", line 30, in __init__
    shard_num=int(data_path), **kwargs
ValueError: invalid literal for int() with base 10: 'data/c1'
```

Hence at the time of creating this PR, I suggested modifying the documentation to remove the "optional" for DATA_PATH. 
We can keep the documentation as is with a more descriptive error message in the example as you suggested in another comment.

Reply:
Reverted change in aae8185f8ab1ba1ce58cd4ff72bc9931b926bb73.